test/e2e/app-dir/dynamic-io/dynamic-io.test.ts

```diff
@@ -168,6 +168,20 @@ describe('dynamic-io', () => {
     }
   )
 
+  itSkipTurbopack(
```

Comment:
why skip turbopack?

Reply:
Just preemptively but I think we can do a clearance now.